spec/factories/strategies/valkyrie_resource.rb

```diff
@@ -1,4 +1,7 @@
-class ValkyrieCreateStrategy
+# This strategy is registered in spec_helper.rb and given a name that can be called by factory bot (e.g. create_using_hyrax_adapter).
+# See spec_helper.rb to confirm registered name.
+# Example:  let(:resource) { FactoryBot.create_using_hyrax_adapter(:hyrax_work) }
+class ValkyrieHyraxAdapterCreateStrategy
```

Comment:
Best not to change this constant name, unless there's a strong motivation for doing so.

Reply:
Is your objection to the rename that this is part of the public API since it is in master?  

I was under the impression, that if there is a reason like readability of code, and it hasn't been part of a release, then it is ok to make the change. 

It is easy enough to change back if in your judgement it should not change.

Reply:
I guess this is subjective, but I prefer the old name.

More generally, I do think that we should err on the side of not renaming existing constants.

Reply:
PENDING response

Reply:
DONE - reverted back to ValkyrieCreateStrategy with a #persister method.  The default persister defined in this class is Hyrax.persister.